File: Nicollas.Dto/Realtime/RealtimeDto.cs
namespace Nicollas.Dto.Realtime
{
    public class RealtimeDto
    {
        public ActionNeeded ActionNeeded { get; set; }
        public string Reducer { get; set; }
        public string Type { get; set; }
        public string ActionName { get; set; }
        public object Result { get; set; }
        public string Method { get; set; }
        public string Callback { get; set; }
    }
}

File: Nicollas.Dto/GeneratedTs/RealtimeDto.ts
/*
* This template uses the TsT - Typewriter.
* Read more at :
* -- https://github.com/frhagn/Typewriter (Github)
* -- https://marketplace.visualstudio.com/items?itemName=frhagn.Typewriter  (Extension of Visual Studio 17)
* -- http://frhagn.github.io/Typewriter/ (Documentation)
*/

/* tslint:disable*/

declare namespace Nicollas.Dto.Realtime {

	interface realtimeDto  { 
        actionNeeded: ActionNeeded;
        reducer: string;
        type: string;
        actionName: string;
        result: System.any;
        method: string;
        callback: string;
    }
}